logstash-core/spec/logstash/queue_factory_spec.rb

```diff
@@ -72,9 +74,10 @@
   context "when `queue.type` is `memory`" do
     before do
       settings.set("queue.type", "memory")
+      settings.set("pipeline.batch.size", 1024)
```

Comment:
no what I mean is that previously the size was not explicitly set in the test, so why do you need to set it now?